src/helpers/hotkeysManager.js

```diff
@@ -330,19 +330,6 @@ WebViewer(...)
 
         print(dispatch, selectors.isEmbedPrintSupported(getState()));
       },
-      [`${Keys.ENTER}`]: () => {
-        if (document.activeElement.className.includes('Note')) {
-          document.activeElement.click();
-        } else if (document.activeElement.className === 'skip-to-document') {
-          document.getElementById('pageText0').focus();
-        } else if (document.activeElement.className === 'skip-to-notes') {
-          dispatch(actions.openElement('notesPanel'));
-          const noteEl = document.querySelector('.Note');
-          if (noteEl) {
-            noteEl.focus();
-          }
-        }
-      },
```

Comment:
This section wasn't needed. Notes are now focusable and keyboard clickable, and the a11y block elements have their own on-click functions.